addons/web_editor/static/src/components/upload_progress_toast/upload_service.js

```diff
@@ -147,7 +147,7 @@ export const uploadService = {
                                 ctx.drawImage(image, 0, 0);
                                 const altDataURL = canvas.toDataURL('image/jpeg', 0.75);
                                 await rpc('/web_editor/attachment/add_data', {
-                                    'name': file.name.replace(/\.webp$/, '.jpg'),
+                                    'name': file.name.replace(/\.webp$/, '.jpg') || '',
```

Comment:
```suggestion
                                    'name': attachment.name.replace(/\.webp$/, '.jpg'),
```
We got an updated name in the attachment when `file.name` is empty.